DxDispatch/src/dxdispatch/DmlDispatchable.h

```diff
@@ -16,11 +34,32 @@ class DmlDispatchable : public Dispatchable
 private:
     std::string m_name;
     std::shared_ptr<Device> m_device;
-    const Model::DmlDispatchableDesc& m_desc;
-    Dispatchable::Bindings m_initBindings;
+    std::variant<Model::DmlDispatchableDesc, Model::DmlSerializedGraphDispatchableDesc> m_desc;
+    Dispatchable::Bindings m_dmlInitBindings;
+    Dispatchable::Bindings m_serializedGraphBindings;
+    bool m_isSerializedGraph = false;
     Microsoft::WRL::ComPtr<IDMLOperator> m_operator;
-    Microsoft::WRL::ComPtr<IDMLCompiledOperator> m_operatorCompiled;
+    Microsoft::WRL::ComPtr<IDMLCompiledOperator> m_compiledOperator;
     Microsoft::WRL::ComPtr<ID3D12Resource> m_persistentBuffer;
     Microsoft::WRL::ComPtr<IDMLBindingTable> m_bindingTable;
     Microsoft::WRL::ComPtr<ID3D12DescriptorHeap> m_descriptorHeap;
+
+    BindPoints m_bindPoints;
+    std::unordered_map<std::string, DML_TENSOR_DATA_TYPE> m_constantDataTypes;
```

Comment:
As we need the constant node data type only during the lifecycle of Initialize method, we should not this variable available at the class level. We should keep local to the Initialize method and pass it to the subsequent helper methods like `GenerateInitialBindingsFromGraph` and `CreateResourceFromConstantNode` from the Initialize method.

Reply:
done